host/discoverd.go

```diff
@@ -105,3 +109,15 @@ func (d *DiscoverdManager) ConnectPeer(ips []string) error {
 	}
 	return err
 }
+
+func (d *DiscoverdManager) UpdateTags(tags map[string]string) error {
+	d.mtx.Lock()
+	defer d.mtx.Unlock()
+	for k, v := range tags {
+		d.inst.Meta[host.TagPrefix+k] = v
```

Comment:
What about using a `-` prefix on a key to indicate that we want to drop it?

In this example:

`flynn-host tags set <hostid> key1=val1,key2=val2,key3=,-key4,-key5=,-key6=val6`

- key1 will be set to value1 (may be added or updated)
- key2 will be set to value2 (may be added or updated)
- key3 will be set to "" (empty) - tag that just "exist"
- key4, key5 and key6 are removed (if existing) - value is simply ignored